integrationtest/osgi/karaf-it/src/test/java/org/hibernate/search/test/integration/osgi/HibernateSearchWithKarafIT.java

```diff
@@ -121,6 +123,9 @@
 				.type( "xml" )
 				.versionAsInProject();
 
+		Properties mavenProperties = new Properties();
+		mavenProperties.load( getClass().getResourceAsStream( "/maven.properties" ) );
```

Comment:
Shouldn't this stream be closed?

Reply:
don't bother, I've changed it locally... testing, if it's green I'll merge.

Reply:
Ha, right, that was dirty work... Thanks :)